internal/scheduler/jobdb/job_run.go

```diff
@@ -188,35 +203,27 @@ func MinimalRun(id uuid.UUID, creationTime int64) *JobRun {
 }
 
 // CreateRun creates a new scheduler job run from a database job run
-func (jobDb *JobDb) CreateRun(
-	id uuid.UUID,
-	jobId string,
-	creationTime int64,
-	executor string,
-	nodeId string,
-	nodeName string,
-	scheduledAtPriority *int32,
-	running bool,
-	succeeded bool,
-	failed bool,
-	cancelled bool,
-	returned bool,
-	runAttempted bool,
-) *JobRun {
+func (jobDb *JobDb) CreateRun(nodeId string, dbRun *database.Run) *JobRun {
```

Comment:
I guess you did this because you had to add a lot more arguments? My concern with this is that the database.Run now leaks into the JobDb.

Reply:
yes, the arguments number was unmanageable after the additional fields I added to `JobRun`. Can elaborate what are the leaks that concerns you exactly ?

Reply:
The leak is having the jobdb package depend on the database package. Ideally the jobdb package does not depend on the mechanism by which we load data from postgres.